### PR TITLE
hw-2

### PR DESCRIPTION
**Pull request by ZhuravskyOleh**
Добрий вечір. Перевірте будьласка)

### CODE REVIEW

File: routes/api/contacts-router.js
Comment (ceREAL133) [inline routes/api/contacts-router.js:1]: All functions in router should be defined and stored in controller file

File: routes/api/contacts-router.js [hunk 1]
```diff
@@ -1,28 +1,90 @@
...
 import contactsService from "../../models/contacts.js";
 
+import { HttpError } from "../../helpers/index.js";
+
 const contactsRouter = express.Router()
 
+const contactAddSchema = Joi.object({
+  name: Joi.string().required().messages({
+    'any.required': `missing required name field`
+  }),
```
Comment (ceREAL133) [inline routes/api/contacts-router.js]: Schema must be defined in schema.js file

File: routes/api/contacts-router.js
Comment (ceREAL133) [inline routes/api/contacts-router.js:1]: Where are middlewares for validating request?

store all logic in one file is a bad practice

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: routes/api/contacts-router.js
Comment (ZhuravskyOleh) [reply routes/api/contacts-router.js:1]: create contacts-controller

File: routes/api/contacts-router.js (hunk 1 above)
Comment (ZhuravskyOleh) [reply routes/api/contacts-router.js]: create schema.js file

File: routes/api/contacts-router.js
Comment (ZhuravskyOleh) [reply routes/api/contacts-router.js:1]: create  middlewares for validating request

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**